Update Preparing_the_two_DataMiner_Agents.md (#5233)
SkylineCommunications/dataminer-docs: dataminer/Administrator_guide/Failover/Configuring_Failover/Preparing_the_two_DataMiner_Agents.md

The primary DMA is a normal DataMiner Agent. In most cases, this will be an existing DMA that is a member of a DMS cluster. It does not require any additional configuration.

> For a brand-new Failover pair where the primary DMA is not yet part of a cluster, the correct procedure is to first add the primary DMA to the cluster using its physical IP. Only after that step should you set up the Failover by adding the backup (empty) DMA.

Author: @MichielMN , you cannot use the Failover UI for a brand new failover pair unless the primary DMA is already part of a cluster. For the use case that I'm trying to document you need to first, add the primary DMA to the cluster, and then create the failover through the UI.

> For a brand-new Failover pair where the primary DMA is not yet part of a cluster, the correct procedure is to first add the primary DMA to the cluster using its physical IP. Only after that step should you set up the Failover by adding the backup (empty) DMA.

Reviewer: @LuisFreitas-Skyline, I know, I object to the use of the word "adding" here as it can be understood as adding the backup agent via the add button on the agents view instead of what you said in your last sentence here. "add the primary DMA to the cluster, and then create the failover through the UI." is much clearer than what's in the PR right now, in my opinion

> For a brand-new Failover pair where the primary DMA is not yet part of a cluster, the correct procedure is to first add the primary DMA to the cluster using its physical IP. Only after that step should you set up the Failover by using the Failover button and adding the backup (empty) DMA.

Reviewer 2: ```suggestion
For a brand-new Failover pair where the primary DMA is not yet part of a cluster, first add the primary DMA to the cluster using its physical IP before you set up Failover.
```

## Backup DataMiner Agent

> The backup DMA must be a newly installed DataMiner Agent.

Reviewer 2: ```suggestion
The backup DMA must be a newly installed DataMiner Agent.

If you are adding a brand-new Failover pair to a cluster, you do not need to add the backup DMA to the cluster yourself. Once the Failover setup is complete, it will be added automatically.
```
